Source/Test/Rewriting/RewritingEngine.cs

```diff
@@ -308,6 +369,7 @@ private void RewriteAssembly(string assemblyPath, string outputDirectory)
                 {
                     // Traverse the assembly to apply each rewriting pass.
                     Debug.WriteLine($"..... Applying the '{pass.GetType().Name}' pass");
+                    pass.AssemblyNameScope = this.AssemblyNameScope;
```

Comment:
Strange, seems noone is using this from the AssemblyRewriter class or subclasses?